src/ensembl/production/djcore/models.py

```diff
@@ -16,25 +16,25 @@
 from django.db import models
 from django.db.models.base import router, NOT_PROVIDED
 from django.template.defaultfilters import truncatechars
+from ensembl.production.djcore.utils import trim_carriage_return
 
-def trim_carriage_return(value):
-    """
-    Remove (and trim) carriage return char from value
-    :param value:
-    :return: trimmed value
-    """
-    return value.replace("\n", " ").replace("\r", " ").replace("  ", " ")
 
 class TrimmedCharField(models.CharField):
     description = "Field automatically replacing carriage returns by spaces"
 
     def to_python(self, value):
         if isinstance(value, str):
             return trim_carriage_return(value)
-        elif value is None:
-            return value
```

Comment:
Not sure why this check was removed, considering line 28, remember that `str(None) == "None"`

Reply:
You're right, may be best to keep it like it was then.